go.local.mod

```diff
@@ -0,0 +1,71 @@
+module github.com/aws-controllers-k8s/rds-controller
```

Comment:
also curious what is the function of this file ?

Reply:
Line 5 of this file indicates that we should use the locally cloned version of runtime instead of the dependency from Github. This makes it easier to debug runtime changes, because I don't need to have the changes committed. I've had this file locally for a while, when debugging `v0.19.0`, so I figured I'd just push these up here in case anyone else needs them. For the most part you won't need to touch these.

Reply:
got it